### DISCUSSION TITLE
Add an option in settings to remove the "Featured" section from the startup.

### DISCUSSION THREAD

**Discussion by extropyst** [KRTirtho/spotube#1048] [Ideas]
This option is full of the most popular genres in the user's area or region, however often the most popular genres are the most hated and smelly, depending on where each one lives (imagine being a Metalhead living in Latin America, where the most popular It's Reggaeton).

 Due to these types of inconveniences, it would be a great advantage if showing that section is optional.

**Comment by R4ZEROMAN** [comment]
yeah seeing the songs that are not liked by on the top of the main screen is horrible for people who don't listen to that type of songs. It irritates people more

**Comment by KRTirtho** [comment]
Customizable/ordering of the sections will be a better sholution I think then just a simple toggle for Featured section.
This way users can,
- Re-order sections
- Add/Remove/Hide them

**Comment by muzzwood** [comment]
> This option is full of the most popular genres in the user's area or region, however often the most popular genres are the most hated and smelly, depending on where each one lives (imagine being a Metalhead living in Latin America, where the most popular It's Reggaeton).
> 
> Due to these types of inconveniences, it would be a great advantage if showing that section is optional.

Yeah I'm surprised this is not already a feature. It was the first thing I looked for after installation. Please add an option to remove both the "featured" and "new releases" section, since they've got nothing to do with the type of music in your library.

**Comment by pikatwinky** [comment]
+1 to this
Yeah, i also wait the hero who finally implements this and release all we live in Latin America from seeing this cancer called reggaeton and Mexican regional music every time we open the app. 
Please
